app/controllers/concerns/client_sortable.rb

```diff
@@ -11,4 +14,23 @@ def clients_sort_order
   def clients_sort_column
     %w[preferred_name id updated_at locale].include?(params[:column]) ? params[:column] : "id"
   end
+
+  def clients_tax_return_status_filter
+    TaxReturnStatus::STATUSES.keys.find { |status| status == params[:status]&.to_sym }
+  end
+
+  def clients_tax_return_stage_filter
+    TaxReturnStatus::STAGES.find { |stage| stage == params[:status] }
+  end
+
+  def filtered_and_sorted_clients(assigned_to: nil)
+    scopes = [
```

Comment:
I'm hoping this creates a pattern we can use to add scopes to the query as we add new filters in future tickets.